streams/readable-streams/pipe-through.js

```diff
@@ -98,11 +98,9 @@ test(() => {
   };
 
```

Comment:
Indeed. It confused me that there are two, and I don't have a clear idea of which tests should go where.